openai/src/commonMain/kotlin/com/xebia/functional/xef/auto/llm/openai/OpenAIRuntime.kt

```diff
@@ -42,9 +38,8 @@ suspend inline fun <reified A> AI<A>.getOrThrow(): A = getOrElse { throw it }
  * @see getOrElse for an operator that allow directly handling the [AIError] case.
  */
 suspend inline fun <reified A> AI<A>.toEither(): Either<AIError, A> =
```

Comment:
Idem: `A` shouldn't be reified.

Reply:
Makes sense. Let's address this on a different PR since they are unrelated.